src/ServiceContainer/SymfonyExtension.php

```diff
@@ -129,6 +140,16 @@ public function process(ContainerBuilder $container): void
     {
     }
 
+    /**
+     * @param ContainerBuilder $container
+     * @param string           $fileName
```

Comment:
Nooo, aligning phpdocs is a nightmare :)

Reply:
yes you're right! a nightmare and useless :)